torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -3347,6 +3347,18 @@ struct to_ir {
         auto kwargs = emitAttributes(apply.attributes());
         return emitForkExpr(apply.range(), forked, args, kwargs);
       }
+      case prim::awaitable: {
+        auto& trees = apply.inputs().tree()->trees();
```

Comment:
Hmm, looks like `tree()` is not guaranteed to be non-null, so this can crash with null pointer deref. Mind checking that it's not nullable? (or if it's a `c10::optional` use `value_or`